modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java

```diff
@@ -50,6 +50,11 @@ public int countByKeywords(long companyId, long groupId, String keywords) {
 		return doCountByKeywords(companyId, groupId, keywords, false);
 	}
 
+	@Override
+	public int countByC_G(long companyId, long groupId) {
+		return countByKeywords(companyId, groupId, null);
```

Comment:
Hey @victorlaerte,
 In this case, you should use the `filterCountByKeywords` method instead and rename the `countByC_G ` method to `filterCountByC_G` too.

 Can you please fix it and resend? Thx :)